lib/routes/exposures/index.js

```diff
@@ -150,54 +150,65 @@ async function exposures(server, options, done) {
     schema: schema.upload,
     handler: async (request, response) => {
       const { id } = request.authenticate()
-      const { exposures, platform, regions, token } = request.body
 
-      await request.verify(token.replace(/-/g, ''))
+      if ('x-chaff' in request.headers === false) {
```

Comment:
We should probably have some random "processing" delay for chaff requests or it may be possible to distinguish based on timing